Assets/Assets/Scripts/Board.cs

```diff
@@ -18,13 +18,18 @@ public class Board : MonoBehaviour
     public GameObject selectedPiecePrefab;
     public GameObject highlightedTilePrefab;
     public CheckersState.State[,] curState;
+    CheckersState.State[,] boardState;
```

Comment:
It doesn't look like these new member variables are used anywhere; if they aren't, then we can get rid of them.